tests/phpunit/tests/test-install.php

```diff
@@ -16,6 +16,10 @@ function test_activate() {
 		$this->assertEquals( POLYLANG_VERSION, $options['version'] );
 	}
 
+	function test_no_uninstall() {
+		$this->assertFalse( defined( 'PLL_REMOVE_ALL_DATA' ), 'The constant PLL_REMOVE_ALL_DATA mut not be defined for this test'  );
```

Comment:
That was indeed the intention. Not as a useful test by itself but as a helper for future tests developments if the developer doesn't initially pay attention to the constant. If you believe that it's useless, then we can remove it.

Reply:
Okay, I think I get it: because of constants being globals, some tests couldn't be performed if the constant is already set.

Maybe it would be clearer to use `PHPUnit::markTestSkipped()` with a corresponding message in the `setUp()` function of this class, like the[ example from PHPUnit documentation.](incomplete-and-skipped-tests-skipping-tests-examples-databasetest-php)